examples/minimal/packages/client-react/vite.config.ts

```diff
@@ -0,0 +1,28 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+
+export default defineConfig({
+  plugins: [react()],
+  server: {
+    port: 3000,
+    fs: {
+      strict: false,
+    },
+  },
+  optimizeDeps: {
+    esbuildOptions: {
+      target: "es2022",
+    },
+  },
+  build: {
+    rollupOptions: {
+      // TODO: revisit this config after splitting out mud config dependencies
+      // from the cli package so we don't need to bundle the cli package
+      external: ["chalk", "locate-path", "path-exists", "find-up"],
+    },
+    target: "es2022",
+  },
+  define: {
+    "process.env": {},
+  },
```

Comment:
tested locally, this doesn't seem to be needed anymore

Reply:
Unrelated to this change, I noticed this error when building the client for production and then trying to send a transaction though:

<img width="476" alt="CleanShot 2023-05-03 at 12 24 54@2x" src="https://user-images.githubusercontent.com/89248902/235879867-a38abb5a-c626-4830-a3e2-7e105da8f475.png">

Reply:
I see that too, will come back to that in a follow up